packages/preview/sns-polylux-template/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name        = "sns-polylux-template"
+version     = "0.1.0"
+entrypoint  = "sns-theme.typ"
+authors     = ["M. Barbieri <https://mbarbieri.it>"]
+license     = "MIT-0"
+description = "Unofficial template for slides in Polylux"
```

Comment:
This description is too broad if your template is designed for a specific university.